corehq/apps/app_manager/translations.py

```diff
@@ -556,6 +556,12 @@ def update_form_translations(sheet, rows, missing_cols, app):
                         value_node.xml.getparent().remove(value_node.xml)
 
     parser = HTMLParser.HTMLParser()
+
+    # Form sheets are escaped on download, but the modules_and_forms sheet,
+    # where form names are updated, is not. Escape form name here so it
+    # surives the subsequent form unescaping.
+    xform.set_name(escape(form.name))
```

Comment:
Interesting. So this is escaped here because other things are? Is making the other things _not_ escaped not an option?